packages/lib/modules/eclp/hooks/EclpChartProvider.tsx

```diff
@@ -25,14 +33,27 @@ export function useEclpChartLogic() {
     isLoading,
   } = useGetECLPLiquidityProfile(pool)
   const theme = useChakraTheme()
-
+  const { theme: nextTheme } = useNextTheme()
   const tokens = useMemo(() => {
     const poolTokens = getPoolActionableTokens(pool).map(token => token.symbol)
 
     return isReversed ? poolTokens.reverse().join('/') : poolTokens.join('/')
   }, [pool, isReversed])
 
-  const secondaryFontColor = theme.semanticTokens.colors.font.secondary.default
+  const secondaryFontColor =
+    nextTheme === 'dark'
```

Comment:
we can probably create a  helper function to select colors that we can use around. Something like:

```
function useSelectColor() {
  const theme = useChakraTheme()
  const { theme: nextTheme } = useNextTheme()
  return (element:string, attr: string) => nextTheme === 'dark'
      ? theme.semanticTokens.colors[element][attr]._dark
      : theme.semanticTokens.colors[element][attr].light ? theme.semanticTokens.colors[element][attr].light
      : theme.semanticTokens.colors.[element].[attr].default
}
```
this will return a function that we can use in most other cases in the app.